reachm-plus.yml

```diff
@@ -83,3 +83,23 @@ nav:
   - Integration with external software:
     - ArduPilot integration: ardupilot-integration.md
     - Bluetooth output and Android mock location: common/tutorials/mock-location.md
+model: M+
+power_port: JST-GH
+index_images:
+  - 'img/reachm-plus/quickstart.png'
```

Comment:
`index_images` shouldn't be here as they have the same name (or can be made into having the same name). You can just use `img/${model}/tutorials.png` img/${model}/reachview.png` et al. in the `index.mdx`

Reply:
Added new variable `model_path` since changing `model` would replace model name in actual docs and renaming directory with images to "M+" is a  bad idea.